starwars/src/App.js

```diff
@@ -9,10 +33,34 @@ const App = () => {
   // side effect in a component, you want to think about which state and/or props it should
   // sync up with, if any.
 
+  const [charactersDataState, setCharactersDataState] = useState([]);
```

Comment:
You must not append State to the state's name, it could just be `charactersData` instead of `charactersDataState`